tests/compat/check.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+f = open('rpc_compat.json')
+compat = json.load(f)
+
+versions = compat['all_versions']
+compat_table = [[True for x in range(len(versions))] for y in range(len(versions))]
+
+for v in versions:
+	print("downloading version ", v, "...")
+	print("downloading corpus for version", v, "...")
+
+for version, not_compat_list in compat['not_compatbile'].items():
```

Comment:
at some point make sure this is formatted with yapf. if you have vtools you can use `task lint:python` otherwise you can run `yapf` manually and there is a formatting style file already in the root of the redpanda tree.